src/media/Sound.js

```diff
@@ -182,21 +176,19 @@
 	*  @param {Function} [options.ready] A function to call when initialization is complete.
 	*  @return {Sound} The new instance of the sound object
 	*/
-	Sound.init = function(options, readyCallback)
-	{
+	Sound.init = function (options, readyCallback) {
 		var appOptions = Application.instance.options;
 
 		// First argument is function
-		if (isFunction(options))
-		{
+		if (isFunction(options)) {
 			options = { ready: options };
 		}
 
 		var defaultOptions = {
-			plugins : appOptions.forceFlashAudio ?
-				[FlashAudioPlugin]:
+			plugins: appOptions.forceFlashAudio ?
+				[FlashAudioPlugin] :
 				[WebAudioPlugin, FlashAudioPlugin],
-			types: ['ogg', 'mp3'],
+			types: ['mp3'],
```

Comment:
Just as a reference for viewers. This is the actual change made. Everything else is versions or syntax. but still worth looking over